src/test/scala/nl/hugo/redbook/ch5/Test5_01.scala

```diff
@@ -1,15 +1,10 @@
 package nl.hugo.redbook.ch5
 
-import org.scalatest._
-
-class Test5_01 extends WordSpec with Matchers {
-  "A Stream" should {
-    "return a list with its contents" in {
-      Stream(1, 2, 3, 4).toList should be(List(1, 2, 3, 4))
-    }
-
-    "return an empty list if it is empty" in {
-      Empty.toList should be(Nil)
-    }
-  }
-}
+import nl.hugo.redbook.ch5.spec.ToListSpec
```

Comment:
It started to avoid replication and improve reusability. In the end I decided to put all tests together, even if they were used only once, rather than having half in a separate file and half not.